packages/shared/src/consts/validIdRegex.ts

```diff
@@ -1,3 +1,3 @@
-export const ALPHANUMERIC_REGEX = /^[a-zA-Z0-9_-]+$/;
+export const ALPHANUMERIC_REGEX = /^[a-zA-Z0-9_:-]+$/;
```

Comment:
what does this change does?

Reply:
Adds the ability to use ":" in te topic and subscriber keys. It's critical, as we suggest user to namespace like this: 'tenan-1234:user-555`